### DISCUSSION TITLE
Inquiry About Adding Asia-Based Nameservers for ngo.us

### DISCUSSION THREAD

**Discussion by sxadxsx** [ngo-us/registry#18] [Ideas]
Firstly, I would like to express my appreciation for the reliable and secure service ngo.us provides. However, I have noticed that the current Nameserver infrastructure for ngo.us does not include servers in Asia, which may result in increased latency for users in this region.

Given the growing number of users and organizations in Asia utilizing ngo.us domains, I am wondering if there are any plans to introduce Nameserver nodes in Asia to optimize DNS resolution times for this region. Expanding coverage to Asia could significantly enhance the user experience and better support the global community of ngo.us users.

If such plans are under consideration, I would love to learn more about the timeline and implementation. Additionally, I am happy to provide any relevant data or feedback from my usage that might help in the decision-making process.

Thank you for your attention, and I look forward to your response.

**Comment by sxadxsx** [comment]
Add he.net dns ?

**Comment by sxadxsx** [comment]
Or consider using cloudflare free plan?
@ngoregistry

**Comment by ngoregistry** [comment]
Thank you for the suggestions. We have added the new nameservers powered by Anycast globally, which should presumably result in improved latency in Asia.
